frontend/src/main/java/puf/frisbee/frontend/view/StartView.java

```diff
@@ -65,6 +65,9 @@ public void init(StartViewModel startViewModel, ViewHandler viewHandler) {
 	}
 
 	@FXML
+//	private void handleLoginRegisterButtonClicked(ActionEvent event) {
```

Comment:
Kann das wieder aus 🤔 ?

Reply:
Das dient nur dazu, direkt in die GameView zu gelangen, also die RegistrationView und die WaitingView beim Testen zu überspringen.

Reply:
Habe es wieder rausgenommen.

Reply:
Für das debugging/testing können wir das schon drin lassen, wir dürfen nur später nicht vergessen, dass wir die auskommentierten Sachen wieder rauswerfen - ich schreib ein Reminder Ticket...